.github/workflows/release-50_publish-docker-manual.yml

```diff
@@ -142,7 +145,7 @@ jobs:
                 tag=$(git name-rev --tags --name-only $(git rev-parse HEAD)) && \
                 [ "${tag}" != "undefined" ] && echo "tag=${tag}" >> $GITHUB_OUTPUT || \
                 echo "No tag, doing without"
-            else
+            elif [ "${type}" == "final" ]; then
```

Comment:
if...elif without else could be risky.
If we ever add a new choice or worst, rename a choice, it would silently ignore it.
It would be safer adding an `else` branch that raises an error. That would force us to cover all cases properly.

Reply:
A swicth/case may be more appropriate here.